src/main/java/games/strategy/engine/framework/map/download/DownloadMapsWindow.java

```diff
@@ -117,13 +120,15 @@ private DownloadMapsWindow(
       progressPanel.download(pendingDownloads);
     }
 
+    pendingDownloads.addAll(ClientContext.DOWNLOAD_COORDINATOR.getDownloads().stream()
```

Comment:
Good catch, I added a filter step to only get successfully terminated, pending and active downloads!